ui/app/components/secret-engine/configure-ssh.hbs

```diff
@@ -16,7 +16,7 @@
           @value={{@model.publicKey}}
           @displayOnly={{true}}
           @allowCopy={{true}}
-          data-test-ssh-input="public-key"
```

Comment:
looking at other areas, it appears that we tend to _not_  include the engine-type in the selector name. Removing 'ssh' here for consistency.